src/GafferOSLModule/GafferOSLModule.cpp

```diff
@@ -60,28 +58,6 @@ using namespace GafferOSL;
 namespace
 {
 
-/// \todo Move this serialisation to the bindings for GafferScene::Shader, once we've made Shader::loadShader() virtual
-/// and implemented it so reloading works in OpenGLShader.
-class OSLShaderSerialiser : public GafferBindings::NodeSerialiser
-{
-
-	virtual std::string postScript( const Gaffer::GraphComponent *graphComponent, const std::string &identifier, const Serialisation &serialisation ) const
-	{
-		const OSLShader *oslShader = static_cast<const OSLShader *>( graphComponent );
-		const std::string shaderName = oslShader->namePlug()->getValue();
-		// We don't serialise a `loadShader()` call for OSLCode nodes
```

Comment:
Given that some folks' gateway into scripting is through examining the contents of .gfr files, I think it'd be nice to continue not to serialise this. So I guess we'd just need an OSLCode serialiser that returns ""?